requirements.txt

```diff
@@ -1,4 +1,5 @@
 qiskit-terra>=0.20.0
+qiskit-algorithms>=0.2.0
 qiskit-optimization>=0.2.0
```

Comment:
It may require qiskit-optimization>=0.6.0 to support qiskit-algorithms.
```suggestion
qiskit-optimization>=0.6.0
```

Reply:
Thanks, yes, I had not changed it yet - one of the todos above in the main comment above noted this was yet to be done. I had intended to do this as I transitioned the notebooks over. That would be the first step, the version update, where they should still work with the new version with qiskit.algorithms, as a compatibility check, then switch to qiskit_algorithms to complete things

Reply:
CI is running here off Opt main branch so pulls in the latest anyway. It all seems to be working so I committed this suggestion first and will make the remaining changes needed for the switch after.